homeassistant/components/bayesian/binary_sensor.py

```diff
@@ -210,7 +210,7 @@ def _async_template_result_changed(event, updates):
             template = track_template_result.template
             result = track_template_result.result
             entity = event and event.data.get("entity_id")
-
+            observation = None
```

Comment:
You can remove this if you want since Python does not do block scoping of variables, or convert it to a type hint:
```py
observation: None
```

Reply:
:exploding_head: removed in 80fd2ff